### ISSUE TITLE
Update meta-viewport rule to be consistent with ACT

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4231]
ACT's updated meta-viewport rule includes a number of edge case not covered by axe-core, see:
https://www.w3.org/WAI/standards-guidelines/act/rules/b4f0c3/proposed/#expectation-1

As a result, things like the following pass axe-core, when they shouldn't:

```html
<meta name="viewport" content="user-scalable=invalid" />
```

```html
<meta name="viewport" content="maximum-scale=invalid" />
```
